pkg/ingester/metrics.go

```diff
@@ -269,6 +266,11 @@ func newIngesterMetrics(
 
 		idleTsdbChecks: idleTsdbChecks,
 
+		openExistingTSDB: promauto.With(r).NewCounter(prometheus.CounterOpts{
+			Name: "cortex_ingester_tsdb_open_duration_seconds",
```

Comment:
By [convention](https://prometheus.io/docs/practices/naming/) counters end with `_total`.
```suggestion
			Name: "cortex_ingester_tsdb_open_duration_seconds_total",
```